packages/strapi-design-system/src/v2/SimpleMenu/SimpleMenu.js

```diff
@@ -6,11 +6,12 @@ import { Typography } from '../../Typography';
 import { Box } from '../../Box';
 import { Flex } from '../../Flex';
 import { Button } from '../../Button';
-import { BaseLink } from '../../BaseLink';
 import { Popover, POPOVER_PLACEMENTS } from '../../Popover';
 import { getOptionStyle } from './utils';
 import { useId } from '../../helpers/useId';
 import { KeyboardKeys } from '../../helpers/keyboardKeys';
+import { Link } from '../../Link';
```

Comment:
we should use the `v2/Link` not the `v1/Link`

Reply:
Noted.